### ISSUE TITLE
why dense_wh got low map?

### ISSUE THREAD

**Issue by YilanWang** [xingyizhou/CenterNet#710]
Thank the great work! Under the same sample method, the 'wh' and 'dense_wh' should get the same result. However, whether the experiments I did or the author did, the MAP is low,   What may be the cause of this?

**Comment by xingyizhou** [comment]
I tried applying regression to the 3x3 grid near the center (but NOT apply the Gaussian weights to normalize them), it actually helps a little bit.

**Comment by YilanWang** [comment]
Yes, I have tried not only Gaussian sampling and single-point sampling, but also sampling at a certain ratio. It is found that the higher the density, the worse the result ... But when only one point is taken, it is also far from the normal setting ... LOL

**Comment by Alien1007** [comment]
@xingyizhou Thanks for your work. Did you set the same regression targets (wh and offset) for those 3 * 3 grids?
